### DISCUSSION TITLE
New Telegram Group

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#1435] [General]
Informo que foi criado um grupo, em português, no Telegram, para discutir o uso do iPED.
O endereço é https://t.me/iped_br

I inform you that a group was created, in Portuguese, on Telegram, to discuss the use of iPED.
The address is https://t.me/iped_br
But if you want to write in English, you will definitely get the answer in English.
I remember that Telegram has very good translation support.

**Comment by lfcnassif** [comment]
Thanks @paulobreim, but I encourage all users that are able to read/write in English to use this Discussion forum, so all knowledge would be centralized in one single indexed place and questions wouldn't be answered twice.

**Comment by lzomatos** [comment]
Hi everyone,
I think that is another address, no?

[]s

Em quarta-feira, 23 de novembro de 2022, Paulo Cesar Breim (PCB) <
***@***.***> escreveu:
